src/lib/containers/CardContainerLogic.tsx

```diff
@@ -41,8 +41,11 @@ export interface CardLink {
   baseURL: string
   // the key that will go into the url
   URLColumnName: string
-  // the column name who's value will be used
+  // the column name who's value will be used as the display text
   matchColumnName: string
+  // If set, use the rowID as the column value.  Otherwise, use the value in the matchColumnName column
+  overrideValueWithRowID?: boolean
```

Comment:
In general this entire interface is confusing to me...I would love if we could come up with a new one that accurately captures the complexity we need. But that's a project for a later day